session_4/blogspot/app/views/articles/_card.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="card mb-2">
+    <div class="card-body">
+        <h3 class="card-title">
+            <%= article.title %>
+        </h3>
+
+        <h5 class="card-subtitle mb-2 text-muted">
+            <%= article.topic %>
+        </h5>
+
+        <p class="card-text">
+            <%= article.content[0, 80] %>[...]
```

Comment:
This would show `[...]` even if the article is less than 80 characters long

Reply:
so should I put a if statement or there's a easy way to do that.. Coz I observed it on iris blogs